src/examples/desktop_opengl.cpp

```diff
@@ -157,13 +161,21 @@ namespace jsar::example
 
         if (arg == "--help")
         {
-          help();
+          help(argv[0]);
```

Comment:
If the argv[0] is a relative or absolute path, please just use the filename only, such as "./path/to/program" => "program"